pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -263,7 +263,7 @@ def distributed_sampler_kwargs(self):
         distributed_sampler_kwargs = dict(num_replicas=self.world_size, rank=self.global_rank)
         return distributed_sampler_kwargs
 
-    def init_optimizers(self, trainer: "Trainer", model: LightningModule) -> Tuple[List, List, List]:
```

Comment:
@Borda I had to remove these trainer references, because flake8 complains Trainer is not imported. But if we import it, we get cyclic import errors. So I don't know a better way.